charts/mlrun-ce/values.yaml

```diff
@@ -405,3 +405,176 @@ kube-prometheus-stack:
     fullnameOverride: state-metrics
   prometheus-node-exporter:
     fullnameOverride: node-exporter
+
+tdengine:
+  enabled: true
```

Comment:
need arch/product to decide if enabled by default

Reply:
Orit said enabled by default